tools/flight-architect.yml

```diff
@@ -0,0 +1,71 @@
+#==============================================================================
+# Copyright (C) 2019-present Alces Flight Ltd.
+#
+# This file is part of OpenFlight Tools.
+#
+# This content and the accompanying materials are made available
+# available under the terms of the Creative Commons
+# Attribution-ShareAlike 4.0 International License which is available
+# at <https://creativecommons.org/licenses/by-sa/4.0/> or alternative
+# license terms made available by Alces Flight Ltd - please direct
+# inquiries about licensing to licensting@alces-flight.com.
+#
+# OpenFlight Tools is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Creative Commons Attribution-ShareAlike
+# 4.0 International License for more details.
+#
+# You should have received a copy of the Creative Commons
+# Attribution-ShareAlike 4.0 International License along with
+# OpenFlight Tools. If not, see:
+#
+#  https://creativecommons.org/licenses/by-sa/4.0/
+#
+# For more information on OpenFlight Tools, please visit:
+# https://github.com/openflighthpc/openflight-tools
+#===============================================================================
+---
+:name: architect
+:url: https://github.com/alces-software/underware/archive/0.6.0-rc4.zip
```

Comment:
This should probably point to the openflighthpc/flight-architect repo.

Reply:
Oops! Had added this before migrating Underware to Flight Architect. Sorted now.